testrtc.py

```diff
@@ -1,27 +1,70 @@
-#!/usr/bin/python2.4
+#!/usr/bin/python
 #
 # Copyright 2014 Google Inc. All Rights Reserved.
 
-"""WebRTC Test
+"""WebRTC
 
-This module serves the WebRTC Test Page.
+This module serves the WebRTC troubleshooter page.
```

Comment:
This is a bit inaccurate.. can you get a better description of what is going on this file?